o8g/scripts/actions.py

```diff
@@ -337,6 +337,7 @@ def flip(card, x=0, y=0):
     card.alternate = ALTERNATE_POLITICAL if not card.alternate else ''
     return
   card.isFaceUp = not card.isFaceUp
+  notify("{} flips {}.".format(me,card))
```

Comment:
Realize what the problem is seeing the function, the card.isFaceUp toggle will have the generic octgn notify so it's ok, but the if check outputs nothing. Could you move the notify to just above the return and phrase it like ```notify("{} turns '{}' to {}".format(me, card, card.alternate if card.alternate else 'Military'))```